<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CombinedAOAI</class>
 <widget class="QDialog" name="CombinedAOAI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Combined AI/AO Application</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="aiTab">
      <attribute name="title">
       <string>Analog Input</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="graphFrame">
         <property name="minimumSize">
          <size>
           <width>660</width>
           <height>340</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lblYCoordinateMax">
           <property name="text">
            <string>10.0V</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>568</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="lblXCoordinateStart">
           <property name="text">
            <string>0Sec</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="lblXCoordinateEnd">
           <property name="text">
            <string>10Sec</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lblYCoordinateMid">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>45</height>
            </size>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::NoSelection</enum>
           </property>
           <property name="flow">
            <enum>QListView::LeftToRight</enum>
           </property>
           <property name="isWrapping" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblDiv">
           <property name="text">
            <string>Div:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sldDiv">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>200</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::NoTicks</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="lblYCoordinateMin">
           <property name="text">
            <string>-10.0V</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkPassThrough">
           <property name="text">
            <string>Pass AI data to AO (latency test)</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnAIStart">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
           <property name="autoDefault">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAIPause">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Pause</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAIStop">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Stop</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="aoTab">
      <attribute name="title">
       <string>Analog Output</string>
      </attribute>
      <widget class="QFrame" name="background">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>391</width>
         <height>451</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame#background{background-image:url(:/AO_AI/Resources/AO.png)}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="BtnSineA">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>124</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>51</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>51</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:url(:/AO_AI/Resources/sine.png)</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="BtnTriangleA">
        <property name="geometry">
         <rect>
          <x>136</x>
          <y>124</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>51</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>51</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:url(:/AO_AI/Resources/triangle.png)</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="BtnSquareA">
        <property name="geometry">
         <rect>
          <x>203</x>
          <y>125</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>51</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>51</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:url(:/AO_AI/Resources/square.png)</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="BtnSineB">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>301</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>51</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>51</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:url(:/AO_AI/Resources/sine.png)</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="BtnTriangleB">
        <property name="geometry">
         <rect>
          <x>139</x>
          <y>302</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>51</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>51</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:url(:/AO_AI/Resources/triangle.png)</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="BtnSquareB">
        <property name="geometry">
         <rect>
          <x>207</x>
          <y>303</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>51</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>51</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:url(:/AO_AI/Resources/square.png)</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="BtnManualB">
        <property name="geometry">
         <rect>
          <x>288</x>
          <y>303</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>51</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>51</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:url(:/AO_AI/Resources/manual.png)</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="BtnManualA">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>125</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>51</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>51</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:url(:/AO_AI/Resources/manual.png)</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtboxHiLevelA">
        <property name="geometry">
         <rect>
          <x>73</x>
          <y>90</y>
          <width>73</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>73</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>73</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtboxLoLevelA">
        <property name="geometry">
         <rect>
          <x>165</x>
          <y>90</y>
          <width>73</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>73</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>73</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>-5</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtboxValueA">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>90</y>
          <width>65</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>2.5</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtboxHiLevelB">
        <property name="geometry">
         <rect>
          <x>73</x>
          <y>269</y>
          <width>73</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>73</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>73</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtboxLoLevelB">
        <property name="geometry">
         <rect>
          <x>165</x>
          <y>267</y>
          <width>73</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>73</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>73</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>-5</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtboxValueB">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>267</y>
          <width>65</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>2.5</string>
        </property>
       </widget>
       <widget class="QLabel" name="chLabelA">
        <property name="geometry">
         <rect>
          <x>268</x>
          <y>34</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 170, 0)</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="chLabelB">
        <property name="geometry">
         <rect>
          <x>271</x>
          <y>211</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 170, 0)</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QSlider" name="timerTrackBar">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>403</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="timerFreqLabel">
        <property name="geometry">
         <rect>
          <x>253</x>
          <y>405</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string>50ms</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnConfigure">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Configure</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>